llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -270,8 +285,26 @@ int main(int argc, char *argv[]) {
     MSG("\n");
   }
 
-  if (ComputeStats)
-    computeStats(*CAS, SummaryIDs);
+  if (ComputeStats.empty()) {
+    return 0;
+  }
+
+  bool PrintToStdout = false;
+  if (StringRef(ComputeStats) == "-")
+    PrintToStdout = true;
+
+  Optional<raw_fd_ostream> CsvStream;
+
+  if (!PrintToStdout) {
+    std::error_code EC;
+    CsvStream.emplace(ComputeStats, EC, sys::fs::OF_None);
+    ExitOnError ExitOnErr;
```

Comment:
A bit confusing to create a new `ExitOnError` here, since there's already one with the same name (`ExitOnErr`) in the same scope. And also, this one seems to lack a banner.

If you don't have a specific reason to create a new one, I suggest using the one that already exists.

Reply:
Marked as unresolved, since it looks like the most recent commit didn't make a change here. Maybe you pushed the wrong one?

Reply:
Sorry about that, it is fixed now